HandEyeCalibration/Transform/Transform.cpp

```diff
@@ -0,0 +1,246 @@
+/*
+Example to show transformations to/from Transformation Matrix
+
+Zivid primarily operate with a (4x4) Transformation Matrix (Rotation Matrix + Translation Vector). 
+This example shows how to use Eigen to convert to and from:
+  AxisAngle, Rotation Vector, Roll-Pitch-Yaw, Quaternion
+
+ It provides convenience functions that can be reused in applicable applications.
+*/
+
+#include <Eigen/Core>
+#include <Eigen/Dense>
+#include <Eigen/Geometry>
+
+#include <opencv2/core/core.hpp>
+#include <opencv2/core/eigen.hpp>
+
+#include <iomanip>
+#include <iostream>
+
+enum class RotationConvention
+{
+    ZYX_Intrinsic,
+    XYZ_Extrinsic,
+    XYZ_Intrinsic,
+    ZYX_Extrinsic,
+    NOF_ROT
+};
+constexpr size_t nofRotationConventions = static_cast<size_t>(RotationConvention::NOF_ROT);
+
+struct RollPitchYaw
+{
+    RotationConvention convention;
+    Eigen::Array3d rollPitchYaw;
+};
+
+struct Representations
+{
+    Eigen::AngleAxisd axisAngle;
+    Eigen::Vector3d rotationVector;
+    Eigen::Quaterniond quaternion;
+    std::array<RollPitchYaw, nofRotationConventions> rotations;
+};
+
+cv::Mat eigenToCv(const Eigen::MatrixXd &);
+Eigen::MatrixXd cvToEigen(const cv::Mat &);
+Eigen::Array3d rotmat2rpy(const Eigen::Matrix3d &rotationMatrix, const RotationConvention &rotation);
+Eigen::Matrix3d rpy2rotmat(const Eigen::Array3d &rollPitchYaw, const RotationConvention &rotation);
+std::string toString(RotationConvention convention);
+Representations transformFromZividToRobot(const Eigen::Affine3d &);
+Eigen::Affine3d transformFromRobotToZivid(const Eigen::Affine3d &, const Representations &);
+
+int main()
+{
+    try
+    {
+        std::cout << std::setprecision(4);
+        std::cout << "This example shows transformations to/from Transformation Matrix" << std::endl;
+
+        std::cout << "Opening .YAML file which contains transformation matrix (PoseState node)" << std::endl;
```

Comment:
![image](https://user-images.githubusercontent.com/46567886/69758396-b7b6d780-115f-11ea-9944-f8abc8f46e45.png)

Add appropriate spacing (new line) between print outs. One after every printout.

Reply:
In an effort to improve the readability of print outs I intentionally reduced the number of empty lines. I kept an empty line between only between different conversions. The first three prints (with numbers) is special, in that they're three prints of the same information.

Reply:
Ok